models/vyos/bgp_address_family/vyos_bgp_address_family.yaml

```diff
@@ -0,0 +1,210 @@
+module: vyos_bgp_family
+version_added: 2.0.1
+short_description: BGP Address Family Resource Module.
+description: 
+- This module manages BGP address family configuration of interfaces on devices running VYOS.
+author: Gomathi Selvi Srinivasan (@GomathiselviS)
+options:
+  config:
+    description: A dict of BGP global configuration for interfaces.
+    type: dict
+    suboptions:
+      as_number:
+        description:
+        - AS number.
+        type: int
+      address_family:
+        description: BGP address-family parameters.
+        type: list
+        elements: dict
+        suboptions:
+          afi:
+            description: BGP address family settings.
+            type: str
+            choices: ['ipv4_unicast', 'ipv6_unicast']
+          aggregate_address:
+            description:
+              - BGP aggregate network.
+            type: list
+            elements: dict
+            suboptions:
+              address:
+                description: BGP aggregate network.
+                type: str
+              as_set:
+                description: Generate AS-set path information for this aggregate address.
+                type: bool
+              summary_only:
+                description: Announce the aggregate summary network only.
+                type: bool
+          network:
+            description: BGP network
+            type: list
+            elements: dict
+            suboptions:
+              address:
+                description: BGP network address
+                type: str
+              path_limit:
+                description: AS path hop count limit
+                type: int
+              backdoor:
+                description: Network as a backdoor route.
+                type: bool
+              route_map:
+                description: Route-map to modify route attributes
+                type: str
+          redistribute:
+            description: Redistribute routes from other protocols into BGP
+            type: list
+            elements: dict
+            suboptions:
+              protocol:
+                description: types of routes to be redistributed.
+                type: str
+                choices: ['connected', 'kernel', 'ospf', 'ospfv3', 'rip', 'ripng', 'static']
+              table:
+                description: Redistribute non-main Kernel Routing Table.
+                type: str
+              route_map:
+                description: Route map to filter redistributed routes
+                type: str
+              metric:
+                description: Metric for redistributed routes.
+                type: int
+      neighbors:
+        description: BGP neighbor
+        type: dict
+        suboptions:
+          address:
```

Comment:
The OpenConfig term here is `neighbor_address`. Can we standardize it across all our models?

Reply:
https://github.com/openconfig/public/blob/master/release/models/bgp/openconfig-bgp-neighbor.yang#L122